Crayon.Console/Program.cs

```diff
@@ -11,12 +11,12 @@ static void Main(string[] args)
             Console.WriteLine("normal");
             Console.WriteLine(Output.BrightBlue($"Bright {Output.Green("and normal green")}"));
 
-            Console.WriteLine(Output.Green($"The difference {Output.Bold("between bold")} and {Output.BrightGreen("bright green")}"));
-            
+            Console.WriteLine(Output.Green($"The difference {Output.Bold("between bold")}, {Output.BrightGreen("bright green")} and {Output.Dim("dim")}"));
```

Comment:
I don't see much difference between normal and dim on my screen but I believe you.